### DISCUSSION TITLE
intercambio de objetos

### DISCUSSION THREAD

**Discussion by azul12345678** [zebnat/morpho-dimension#31] [Ideas]
me pareceria bueno poder intercambiar objetos que nos salen cuando jugamos y ya lo teníamos por ejemplo, poder hacer un tipo de mercado para intercambiar prendas de avatares repetidos con otros usuarios de la comunidad o poder vender las prendas repetidas por un precio menor para otros usuarios, ayudaría un poco a los recién llegados que no pueden comprar las prendas para sus avatares por tener poca plata, ya que al principio cuesta ganar dinero, entonces como que eso los ayudaría un poco a los recién llegados

**Comment by DarkPegassy** [comment]
Si, o también poder regalar objetos a tus amigos, oh a las persona que elijas en general.
